services/notification/client/client.go

```diff
@@ -85,3 +111,30 @@ func (c *Client) GetParsedResponse(ctx context.Context, method, path string, que
 
 	return resp, errors.WithStack(d.Decode(obj))
 }
+
+type GetProjectRunWebhookDeliveriesOptions struct {
+	*ListOptions
+
+	StartSequence uint64
+}
+
+func (o *GetProjectRunWebhookDeliveriesOptions) Add(q url.Values) {
+	o.ListOptions.Add(q)
+
+	if o.StartSequence > 0 {
+		q.Add("startsequence", strconv.FormatUint(o.StartSequence, 10))
+	}
+}
+
+func (c *Client) GetProjectRunWebhookDeliveries(ctx context.Context, projectID string, deliveryStatusFilter []string, opts *GetProjectRunWebhookDeliveriesOptions) ([]*types.RunWebhookDelivery, *Response, error) {
```

Comment:
shouldn't deliveryStatusFilter be part of GetProjectRunWebhookDeliveriesOptions?